### PR TITLE
Sample data

### PR DESCRIPTION
**Pull request by chrisbrainerd**
The demo page of kepler.gl has a _Try Sample Data_ option which is in one of the demo pages, but not exposed in the actual kepler.gl library.

Being able to quickly and easily see the value of kepler.gl is made much easier with sample data, and users using this via Atlas might not have data ready to go, so keeping this option in production is still useful. 

This PR adds a prop to the top level component `dataSamples` to pass in your own samples. 

See [Atlas-kepler PR](https://github.com/mapbox/atlas-kepler/pull/8) for example usage, this PR can be linked to that repo and seen working.

### CODE REVIEW

**Comment by johnk1** [comment]
All these changes are going to make it hard to stay in sync with upstream/master. Can we reduce them? At least formatting changes could be undone. In the end, if we are making valuable additions to kepler.gl we should then endeavor to push them back to keplergl/kepler.gl. Maybe one day we can get rid of our fork.

Also, we should target a different branch, I think, rather than master. Again so we can easily bring in upstream/master changes. Maybe the `no_html_export` branch although with these changes that name is no longer accurate. Maybe just `mbx_master` or something.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "kepler.gl",
+  "name": "@mapbox/kepler.gl",
   "author": "Shan He <shan@uber.com>",
-  "version": "1.1.7",
+  "version": "1.2.2",
```
Comment (johnk1) [inline package.json]: before we can publish this we'll need to bump the ver

**Comment by chrisbrainerd** [comment]
> Also, we should target a different branch, I think, rather than master. Again so we can easily bring in upstream/master changes. Maybe the no_html_export branch although with these changes that name is no longer accurate. Maybe just mbx_master or something.

Hm, what's the value in keeping our own copy of `master`? To me, we should keep our fork's master on the cutting edge of whatever our changes are. Bringing in upstream changes from their `master` to our `master` doesn't really accomplish much if we aren't using our `master` for anything. 

If we keep our `master` as the head we're merging our PRs in, it still shouldn't be too bad to merge in upstream changes, we just PR from their master to ours (and optionally make a branch to resolve conflicts if we need to and PR from that).

**Comment by chrisbrainerd** [comment]
Also I can undo the formatting, but the formatting is being done according to the repo's own settings :/ (https://github.com/mapbox/kepler.gl/blob/master/.editorconfig https://github.com/mapbox/kepler.gl/blob/master/.eslintrc https://github.com/mapbox/kepler.gl/blob/master/.prettierrc).

_OR_, and hear me out here - we fix it upstream 🙃 
~https://github.com/keplergl/kepler.gl/pull/753~ https://github.com/keplergl/kepler.gl/pull/334

But point being that this is being fixed so _in theory_ at least, if we start running prettier on this fork, it will actually converge with upstream once that PR gets in.

**Comment by johnk1** [comment]
> If we keep our `master` as the head we're merging our PRs in, it still shouldn't be too bad to merge in upstream changes, we just PR from their master to ours (and optionally make a branch to resolve conflicts if we need to and PR from that).

I agree with that.

> But point being that this is being fixed so in theory at least, if we start running prettier on this fork, it will actually converge with upstream once that PR gets in.

True, and I wonder if the prettier PR was waiting on (at least partially) a new layer type they just merged in. Maybe the time is right to pester them about it again :) However, until they do it, won't including those changes make it potentially more difficult to merge upstream master branch change if / when we want them? We'll more likely end up w/conflicts due to formatting mismatch.

**Comment by chrisbrainerd** [comment]
Reverted formatting changes and merged in master / fixed conflicts

**Review by johnk1 — APPROVED** [review]
🎸

**Pull request merged**